nengo/spa/actions.py

```diff
@@ -154,7 +154,11 @@ class Actions(object):
     def __init__(self, *args, **kwargs):
         self.actions = None
         self.args = args
-        self.kwargs = kwargs
+        self.kwargs = OrderedDict(kwargs)
```

Comment:
Because dicts are handled differently in Python 2 and Python 3, I used an OrderedDict so that I could easily write a tests that would pass for both and so that the outcome would be the same regardless of Python version